docs/rfcs/751-write-empty-file-behavior.md

```diff
@@ -0,0 +1,51 @@
+- Author:  Abyss-w <mad.hatter@foxmail.com>
+- Start Date: 2021-09-10
+- RFC PR: [beyondstorage/go-storage#751](https://github.com/beyondstorage/go-storage/pull/751)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+
+# GSP-751: Write Empty File Behavior
+

```

Comment:
How about adding [Specify the behavior for writing empty file](https://forum.beyondstorage.io/t/topic/204) as the previous Discussion?

Reply:
I think it can be done like this.